docs/source/contribution/development_for_databricks.md

```diff
@@ -0,0 +1,89 @@
+# How to deploy a development version of Kedro to Databricks
+
+[Databricks](https://www.databricks.com/) is a cloud-based platform for data engineering and data science that many Kedro users deploy their Kedro projects to. Extending and improving the Kedro experience on Databricks is important to a large part of Kedro's user base.
+
+## Introduction
+
+```{note}
+This page is for people developing Kedro, users working on their own Kedro projects should see the documentation for [deploying Kedro projects on Databricks](../deployment/databricks.md).
+```
+
+This guide describes how to efficiently develop features and fixes for Kedro on Databricks. Using this guide, you will be able to quickly test your locally modified version of Kedro on Databricks as part of a build-and-test development cycle.
```

Comment:
I'd be tempted to move these two sentences to precede the note rather than follow it, but will leave it up to you.

Reply:
That's fair, it's probably bad form to start a section with a note. Fixed this.